frontend/src/lib/taxonomy.tsx

```diff
@@ -1076,6 +1076,11 @@ export const CORE_FILTER_DEFINITIONS_BY_GROUP = {
         },
     },
     replay: {
+        snapshot_source: {
+            label: 'Platform',
+            description: 'Platform the session was recorded on',
+            examples: ['web', 'mobile'],
```

Comment:
I wonder if this could confuse folks searching for replays recorded on mobile via a web browser, they have to use `web` in this case but it's counterintuitive, how could we make that clearer?

Reply:
Reasonable question across the products now... how do we distinguish this concept in general 🤔 

We could say "SDK" and "SDK this was recorded with" 

But then maybe "snapshot_source" should start to say "android" and "ios" when sent instead of "mobile"

Reply:
I think snapshots should set `$device_type`, and some extra metadata such as `$os_*`, `snapshot_source` is still fine because this is more about the wireframes approach, I'd not mix it up with where it was recorded but just the how.

Reply:
If you search for Device type=Mobile, you see events and recordings recorded on native mobile or web.
If you search for OS=Android, you see events and recordings recorded on native only.

Reply:
Agreed, this is a little confusing. Going to push forward with this PR because it's really only a visual change once we make a decision on what we'd like to name things. This way we'll learn if it does confuse customers.

Could be a good candidate discussion for making mobile a first class citizen - it's these kinds of things that people don't consider when building